test/runtests.jl

```diff
@@ -95,6 +96,7 @@ T = Float32
         @test isapprox( signalsum, T(2), atol = 5e-3 )
         nt = NamedTuple(sim)
         @test sim == Simulation(nt)
+        @test isapprox(get_depletion_voltage(sim, 1, verbose = false), T(99), atol = 0.2) #13.5V depletion voltage was simulated before. This makes sure code changes do not affect the computation.
```

Comment:
I am a bit surprised about this one because the `BEGe` has no impurity density defined.
I would expect the depletion voltage of this detector to be zero...

Two options:
- Easy fix for this: can you delete this test?
- More work: can you look into what happened here? Why isn't the depletion voltage zero?

Reply:
I will look into this. I was wondering what was happening here too

Reply:
Turns out, that the code could not figure out a depletion voltage of 0, since the criterium in the global search is that max-min < applied voltage. If the applied voltage is 0, the noisy left side will always be bigger. I changed the check to maximum(applied_voltage, eps) where eps=1e-3 to fix this